tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,119 @@
+
+get_global_transform_interpolated()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What we really want to focus the Camera on, is not the position of the target on the physics tick, but the *interpolated* position, i.e. the position at which the target will be rendered.
+
+We can do this using the :ref:`Spatial.get_global_transform_interpolated<class_Spatial_method_get_global_transform_interpolated>` function. This acts exactly like getting :ref:`Spatial.global_transform<class_Spatial_property_global_transform>` but it gives you the *interpolated* transform (during a ``_process()`` call).
+
+.. important:: ``get_global_transform_interpolated()`` should only be used once or twice for special cases such as Cameras. It should **not** be used all over the place in your code (both for performance reasons, and to give correct gameplay).
+
+.. note:: Aside from exceptions like the Camera, in most cases your game logic should be in ``_physics_process()`` and you should be calling ``get_global_transform()`` or ``get_transform()``, which will give the current physics transform (in global or local space respectively), which is usually what you will want for gameplay code.
+
+Example manual Camera script
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is an example of a simple fixed Camera which follows an interpolated target:
+
+.. code-block:: python
+
+	extends Camera
+		
+	# Node that the camera will follow
+	var _target
+		
+	# We will smoothly lerp to follow the target
+	# rather than follow exactly
+	var _target_pos : Vector3 = Vector3()
+		
+	func _ready() -> void:
+		# Find the target node
+		_target = get_node("../Player")
+		
+		# Turn off automatic physics interpolation for the Camera,
+		# we will be doing this manually
+		set_physics_interpolated(false)
+		
+	func _process(delta: float) -> void:
+		# Find the current interpolated transform of the target
+		var tr : Transform = _target.get_global_transform_interpolated()
+		
+		# Provide some delayed smoothed lerping towards the target position 
+		_target_pos = lerp(_target_pos, tr.origin, min(delta, 1.0))
+		
+		# Fixed camera position, but it will follow the target
+		look_at(_target_pos, Vector3(0, 1, 0))
+
+Mouse look
+^^^^^^^^^^
+
+Mouse look is a very common way of controlling Cameras. But there is a problem. Unlike keyboard input, the mouse is not polled at each physics tick. New mouse move events come in continuously, and the Camera will be expected to react and follow these on the next frame, rather than waiting until the next physics tick.
+
+In this situation it can be better to disable physics interpolation for the Camera node (using :ref:`Node.set_physics_interpolated<class_Node_method_set_physics_interpolated>`) and directly apply the mouse input to the Camera rotation, rather than apply it in ``_physics_process``.
```

Comment:
```suggestion
In this situation, it can be better to disable physics interpolation for the Camera node (using :ref:`Node.set_physics_interpolated<class_Node_method_set_physics_interpolated>`) and directly apply the mouse input to the Camera rotation, rather than apply it in ``_physics_process``.
```